internal/core/tests.rs

```diff
@@ -84,47 +84,21 @@ pub extern "C" fn send_keyboard_string_sequence(
 }
 
 cfg_if::cfg_if! {
-    if #[cfg(target_arch = "wasm32")] {
-        use wasm_bindgen::prelude::*;
-
-        #[wasm_bindgen]
-        extern "C" {
-            #[wasm_bindgen(js_namespace = console)]
-            pub fn log(s: &str);
-        }
 
+    if #[cfg(feature = "std")] {
```

Comment:
Why is this still in a cgf_if ?   the two branch looks the same to me.

Reply:
The difference is the use of the trait for the alloc crate. With std that doesn’t compile out of the box but requires the extern crate alloc on all call sites. That would be a new and annoying requirement IMO

Reply:
You can do something like `use $crate::testing::ToString;`   where ToString is a re-export.